mypy/main.py

```diff
@@ -509,6 +521,9 @@ def add_invertible_flag(flag: str,
         '--baseline-file', action='store',
         help="Use baseline info in the given file"
              "(defaults to '{}')".format(defaults.BASELINE_FILE))
+    add_invertible_flag(
+        '--no-auto-baseline', default=True, group=based_group,
+        dest="auto_baseline", help="Don't update the baseline automatically.")
```

Comment:
i think it should auto baseline by default, also the double negative makes it confusing
```suggestion
    add_invertible_flag(
        '--auto-baseline', default=True, group=based_group,
        dest="auto_baseline", help="update the baseline automatically.")
```

Reply:
This only appears backwards, it functions are you are requesting.